src/connectors/argent/argentMobile/index.ts

```diff
@@ -95,8 +96,14 @@ export class ArgentMobileBaseConnector extends Connector {
     return this._wallet
   }
 
-  async connect(): Promise<ConnectorData> {
-    await this.ensureWallet()
+  async connect(
+    props:
```

Comment:
does this cause any conflicts with the `Connector` interface when used with starknet-react? (typescript issue basically)